charts/values.example.mla.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
```

Comment:
@xrstf I couldn't find where the magic happens - how do I make sure that this values file will be included in the installation package in `examples` directory?